junit-jupiter-engine/src/test/java/org/junit/jupiter/engine/execution/ParameterResolutionUtilsTests.java

```diff
@@ -137,9 +136,9 @@ void onlyConsiderParameterResolversThatSupportAParticularParameter() {
 		thereIsAParameterResolverThatDoesNotSupportThisParameter();
 		thereIsAParameterResolverThatResolvesTheParameterTo("something");
 
-		invokeMethod();
+		Object[] arguments = resolveMethodParameters();
 
-		verify(instance).singleStringParameter("something");
+		assertThat(arguments).hasSize(1).containsExactly("something");
```

Comment:
```suggestion
		assertThat(arguments).containsExactly("something");
```

`hasSize()` is already covered by `containsExactly()`